src/h5.cpp

```diff
@@ -245,14 +242,14 @@ append_ret h5d::append_data_1d(T const *data, hsize_t nlen) {
   try {
     Dataset.write(*data, Type, DSPMem, DSPTgt, PLTransfer);
   } catch (...) {
-    LOG(Sev::Error, "write failed  ds_name: {}", ds_name);
-    if (log_level >= 7) {
+    if (Logger->should_log(spdlog::level::trace)) {
```

Comment:
```suggestion
    Logger->error("write failed  ds_name: {}", ds_name);
    if (Logger->should_log(spdlog::level::trace)) {
```

Reply:
The suggested change does not look like it should here. The thing is that  the _write failed_ message was changed from being an error to being a a trace message here and it was moved from being outside the if-statement to being inside it. I moved it back outside and changed the severity back into _error_.